README.md

```diff
@@ -102,6 +102,7 @@ Additional lists you might find useful:
 - [CookieAuth plugin](https://github.com/Xety/Cake3-CookieAuth) - A simple Cake 3 plugin to automatically authenticate users with Cookies.
 - [HierAuth plugin](https://github.com/btaens/cakephp-hier-auth) - A CakePHP plugin for hierarchical, role based, simple authorization.
 - [Muffin/OAuth2 plugin](https://github.com/usemuffin/oauth2) - OAuth2 authentication using the [`league/oauth2-client`](https://github.com/thephpleague/oauth2-client).
+- [Muffin/Tokenize plugin](https://github.com/UseMuffin/Tokenize) - Security tokens for CakePHP 3.
```

Comment:
You are right, I didn't think, I will try to come up with something respecting the guidelines